src/idpf.rs

```diff
@@ -135,22 +136,70 @@ where
 /// Trait for values to be programmed into an IDPF.
 ///
 /// Values must form an Abelian group, so that they can be secret-shared, and the group operation
-/// must be represented by [`Add`]. An implementation of [`CoinToss`] must be provided to randomly
-/// select a value using PRG output. Values must be encodable and decodable, without need for a
-/// decoding parameter.
+/// must be represented by [`Add`]. Values must be encodable and decodable, without need for a
+/// decoding parameter. Values can be pseudorandomly generated, with a uniform probability
+/// distribution, from PRG output.
 pub trait IdpfValue:
     Add<Output = Self>
     + AddAssign
     + Sub<Output = Self>
-    + ConditionallySelectable
     + ConditionallyNegatable
-    + CoinToss
     + Encode
     + Decode
     + Sized
 {
+    /// Any run-time parameters needed to produce a value.
+    type Parameters;
```

Comment:
* nit: s/Parameters/Parameter for consistency with the codec traits
* Suggestion: A more specific sounding name might make the trait more self-documenting. Perhaps `OutputParameter` or `ValueParameter`?